packages/mdc-checkbox/_mixins.scss

```diff
@@ -219,7 +216,7 @@ $mdc-checkbox-ripple-target: ".mdc-checkbox__ripple";
 ///
 /// Sets density scale for checkbox.
 ///
-/// @param {Number | String} $density-scale - Density scale value for component. Supported density scale values `-4`,
+/// @param {Number | String} $density-scale - Density scale value for component. Supported density scale values
```

Comment:
Should this just support number?

Reply:
It can technically accept `maximum`, `minimum` or `default` literals. But numbers are preferred.